### ISSUE TITLE
webpage preview

### ISSUE THREAD

**Issue by DarkknightAK** [nextcloud/deck#1702]
Hi,

would it be possible to get an link preview running like you know it from chat applications you post an link (to a youtube movie) and see instead of the link an preview where you can click on it.

**Comment by juliushaertl** [comment]
Makes sense, probably once we have integrated the Nextcloud Text editor with #1663

**Comment by jakobroehrl** [comment]
The Text issue for this topic: https://github.com/nextcloud/text/issues/683
